airflow/www/static/css/bootstrap-theme.css

```diff
@@ -5945,6 +5945,12 @@ button.close {
 .affix {
   position: fixed;
 }
+.truncate-text {
```

Comment:
@vchiapaikeo This doesn't look like the right file to host this CSS since it's Bootstrap's CSS if we upgrade we are likely to change this file. 

Can we please move it to `www/static/css/main.css` instead?

Reply:
Ooo right, good call! I want to say that I tried this but this stylesheet wasn't being used in this view. In any case, let me get rid of this altogether because Elad's suggestion to check out `VariableModelView` that leverages `list_template` looks a lot cleaner!